app/steps/ui/summary/template.html

```diff
@@ -0,0 +1,28 @@
+{% extends "includes/two_thirds_form.html" %}
```

Comment:
`{% extends "includes/layout.html" %}`

Reply:
Hi, I should have said to you that I've changed the summary page processing to introduce 'post' handling that does not happen in the PA journey.  The main template for including post functionality is the standard two_thirds_form.html, i discussed the issue with David and rather than duplicating the basic template or adding form statements to the page , he suggested that i made the two_thirds_form.html dynamic by passing in the class name required for the width. I think with hindsight that i should perhaps change the name of the template to 'dynamic_form.html'  or similar?

Reply:
Right, makes sense, yeah maybe duplicate `two_thirds_form.html` and call it `full_width_form.html`?

Reply:
David was unhappy with me doing that and that's why i changed the two_thirds_form.html to accept an override property with a value of 'column_full', if the property is not set it defaults to 'column_two_thirds'.  He was against duplicating the files with only a class name difference between them.